templates/docs/examples/patterns/logo-section/logo-section-dark.html

```diff
@@ -3,15 +3,7 @@
 
 {% block standalone_css %}patterns_logo-section{% endblock %}
 
-{% block style %}
-<style>
-body {
-  background: #262626;
-  color: #FFF;
-}
-</style>
-{% endblock %}
-
+{%set is_dark = True%}
```

Comment:
Just a bit of missing whitespace for consistency 
```suggestion
{% set is_dark = True %}
```